packages/docs/src/examples/Transaction.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { Mainnet, DAppProvider, useSendTransaction, useEthers, Config, ChainId } from '@usedapp/core'
+
+const config: Config = {
+    readOnlyChainId: Mainnet.chainId,
+    readOnlyUrls: {
+      [Mainnet.chainId]: 'https://mainnet.infura.io/v3/57fc2c19095745e59ab96a4aa87dada8',
+    },
+}
+
+ReactDOM.render(
+    <DAppProvider config={config}>
+      <App />
+    </DAppProvider>,
+    document.getElementById('root')
+)
+
+export function App() {
+    const { chainId } = useEthers()
+    const { sendTransaction, state } = useSendTransaction()
+    
+    // We prevent the example from running on Mainnet so that the users do not use real Ether without realizing.
+    const disabled = chainId === ChainId.Mainnet
+    const status = state.status
+    const address = '0xe13610d0a3e4303c70791773C5DF8Bb16de185d1'
+
+    const send = () => {
+        sendTransaction({ to: address, value: 1 })
```

Comment:
This is one `wei`, not `Gwei` :D So either change it here or in the description of the example.

Reply:
:)